### PR TITLE
[Editted] SQL Commands:ORDER BY

### PR DESCRIPTION
**Pull request by kevuschka**
### Description

Closes issue #3346

The PR belongs to the issue #3346
I added some more lines to the existing SQL order-by.md.
My addings include some lines to mention the possibility to ORDER BY (or sort) multiple columns, if they exist in that SQL table.

I am learning with the "Learn Git & Github" course from Codecademy, that's where my motivation comes from.
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the 
"Description" above).

### CODE REVIEW

File: content/sql/concepts/commands/terms/order-by/order-by.md [hunk 1]
```diff
@@ -25,6 +25,15 @@ SELECT column_name
 FROM table_name
 ORDER BY column_name ASC | DESC;
 ```
+It's also possible to `ORDER BY` **multiple columns**: 
+> _by seperating with a comma_
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/order-by/order-by.md:29]:
```suggestion

It's also possible to `ORDER BY` multiple columns by separating them with a comma.

```

File: content/sql/concepts/commands/terms/order-by/order-by.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ SELECT column_name
...
+```
+SELECT column_name_1
+FROM table_name
+ORDER BY column_name_1 ASC | DESC, column_name_2 ASC | DESC;
+```
+-  assuming **_column_name_1_** and **_column_name_2_** exist in **table_name**
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/order-by/order-by.md:35]:
```suggestion

This assumes `column_name_1` and `column_name_2` exist in `table_name`.
```

File: content/sql/concepts/commands/terms/order-by/order-by.md [hunk 3]
```diff
@@ -25,6 +25,15 @@ SELECT column_name
...
+
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/order-by/order-by.md:36]:
```suggestion
```
Remove blank line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kevuschka I did a first review and left a few comments.

**Comment by kevuschka** [comment]
@SSwiniarski I commited your suggestions directly here on github

**Review by SSwiniarski — APPROVED** [review]
@kevuschka Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@kevuschka @SSwiniarski LGTM

**Pull request merged**